src/appmixer/trello/trello-commons.js

```diff
@@ -32,7 +29,7 @@ module.exports = {
             await context.sendJson(records, outputPortName);
         } else if (outputType === 'file') {
             // Into CSV file.
-            const headers = Object.keys(records[0]);
+            const headers = Object.keys(records[0] || {});
```

Comment:
This fixes `Error: Cannot convert undefined or null to object` when there are no `records`.